api/v1/storagecluster_types.go

```diff
@@ -109,22 +109,6 @@ type StorageClusterSpec struct {
 	// DefaultStorageProfile is the default storage profile to use for
 	// the storageclassrequest as StorageProfile is optional.
 	DefaultStorageProfile string `json:"defaultStorageProfile,omitempty"`
```

Comment:
@Madhu-1 What happens if more then 1 profile is marked with the annotation?

Reply:
if we have one or more profile, we need to take a decision on what to do, i was comparing this to the storageclass, If we have 2 default storageclasses, nothing will be chosen when PVC is created without storageclass.

Reply:
@Madhu-1 Why push ourselves into that corner when we can just have the default be selected by the storage cluster without ambiguity?

Reply:
That's true, I was trying to keep both resources independent. If something needs to be set to default it need to be updated in the storageCluster CR not on the StorageProfile CR, this can be marked as resolved

Reply:
Glad we got to a resolution